SurveyAppTests/Tests/Modules/Login/LoginRouterSpec.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  LoginRouterSpec.swift
+//  SurveyApp Tests
+//
+//  Created by Chananchida F. on 6/17/21.
+//
+
+import Quick
+import Nimble
+
+@testable import SurveyApp
+
+final class LoginRouterSpec: QuickSpec {
+
+    override func spec() {
+        var router: LoginRouter!
+        var viewController: LoginViewController!
+        var window: UIWindow!
+
+        describe("a LoginRouter") {
+            beforeEach {
+                viewController = LoginViewController()
+                viewController.output = LoginViewOutputMock()
+
+                router = LoginRouter()
+                router.view = viewController
+                _ = viewController.view
+            }
+
+            context("when showHome is called") {
```

Comment:
I think we should use `describe` here 
```suggestion
            describe("its showHome is called") {
```

We're trying to follow these rules, right?
```
describe("its call something")
  context("when in condition A")
  context("when in condition B")
  ...
```

Reply:
@markgravity In this case there is no condition, so should it be like this?
```Swift
describe("its showHome is called") {
    beforeEach {
        ...
    }

    it("shows a HomeModule view") {
        ...
    }
}
```